### DISCUSSION TITLE
Custom Models - Bird Classifcation

### DISCUSSION THREAD

**Discussion by tmuellerchen** [blakeblackshear/frigate#2413] [Ideas]
First: Thanks a lot for this great software. I love it :)

Right now I use it for my birdfeeder in my garden. 
Is it possible to use a custom model like this: 
https://tfhub.dev/google/lite-model/aiy/vision/classifier/birds_V1/3

with Frigate?

**Comment by ozett** [comment]
its worked on:
https://github.com/blakeblackshear/frigate/issues/2574
https://github.com/snowzach/doods2/issues/3#issuecomment-1004828461

not closely integrated, but if the doods2 developer builds it freshly into doods2, you than could feed images into it..
